src/Validator/Constraints/ExpressionValidator.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Overblog\GraphQLBundle\Validator\Constraints;
+
+use Overblog\GraphQLBundle\Validator\ValidationNode;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Constraints\Expression;
+use Symfony\Component\Validator\Exception\LogicException;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+class ExpressionValidator extends \Symfony\Component\Validator\Constraints\ExpressionValidator
+{
+    private $expressionLanguage;
+
+    public function __construct(ExpressionLanguage $expressionLanguage = null)
+    {
+        $this->expressionLanguage = $expressionLanguage;
+        parent::__construct(null, $expressionLanguage);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function validate($value, Constraint $constraint): void
+    {
+        if (!$constraint instanceof Expression) {
+            throw new UnexpectedTypeException($constraint, __NAMESPACE__.'\Expression');
+        }
+
+        $variables = $constraint->values;
+        $variables['value'] = $value;
+
+        $object = $this->context->getObject();
+
+        $variables['this'] = $object;
+
+        if ($object instanceof ValidationNode) {
+            $variables['prevValue'] = $object->getResolverArg('value');
```

Comment:
`rawValue` instead of  `prevValue`?

Reply:
No, `rawValue` would rather mean an "unprepared" or an "unprocessed" value. But the "prevValue" represents the value of the "previous" resolver or "parent" resolver. So the names should be `prevValue` or `parentValue`.

Reply:
ok for `parentValue`

Reply:
Done!